Tabloid/client/src/components/Header.js

```diff
@@ -18,36 +18,53 @@ export default function Header({ isLoggedIn }) {
   return (
     <div>
       <Navbar color="light" light expand="md">
-        <NavbarBrand tag={RRNavLink} to="/">Tabloid</NavbarBrand>
+        <NavbarBrand tag={RRNavLink} to="/">
+          Tabloid
+        </NavbarBrand>
         <NavbarToggler onClick={toggle} />
```

Comment:
what is `toggle`?

Reply:
Toggle allows for the navbar to be open and closed when the page is shrunk. @heymonicakay